packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue

```diff
@@ -0,0 +1,395 @@
+<template>
+
+  <ImmersivePage
+    :appBarTitle="$tr('toolbarHeader')"
+    :route="backRoute"
+    :icon="icon"
+  >
+    <KPageContainer v-if="device" :style="pageHeight">
+      <KGrid gutter="48" class="edit-sync-schedule">
+
+        <KGridItem>
+          <h1>{{ $tr('editSyncScheduleSubTitle') }}</h1>
+        </KGridItem>
+
+        <KGridItem>
+          <p>{{ device.device_name }} </p>
+        </KGridItem>
+
+        <KGridItem class="select-element">
+          <div
+            class="k-select"
+          >
+            <KSelect
+              v-model="selectedInterval"
```

Comment:
`selectedInterval` is not defined anywhere, so this is giving an error on render.

Note that the value passed here as the v-model should correspond to one of the `options` (or one of the value properties of one of the options) being passed in from `selectArray`, or `null`, as you have done with `selectedDay` and `selectedTime`.